README.rst

```diff
@@ -0,0 +1,23 @@
+GENETIS_HPol
+============
+
+The genetic algorithm (GA) is a submodule 
+(links to Shared-Code/GA; branch HPol; commit "HPol pause")
+
+
+GENETIS_doc
+-----------
+This directory contains the documentation.
+
+To view the documentation, please use this `link
+<https://genetis-doc.vercel.app/>`_.
+
+To edit the `rst` files locally with Sphinx, note that 
+
+1.  ``furo`` and ``sphinxnotes.strike`` are used so you'd need to 
+    ``pip install furo`` and ``pip install sphinxnotes-strike``
+2.  ``dropdown`` menu is used, so please ``pip install sphinx-panels``.
+3.  ``autodoc`` is used for automatic python documentation and ``todo`` is used
+    for to-do lists (no need to install these two)
+
+(see ``docs/source/conf.py``), 
```

Comment:
```suggestion
(see ``docs/source/conf.py``)
```

Watch out for trailing whitespace -- configure your editor to automatically remove them on save.